shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -3214,3 +3214,193 @@ forRefreshClause
+    ;
+
+storageMaxSizeClauses
+    : (MAXSIZE | MAX_AUDIT_SIZE | MAX_DIAG_SIZE) (UNLIMITED | sizeClause)
+    ;
+
+pdbLoggingClauses
+    : loggingClause | pdbForceLoggingClause
+    ;
+
+pdbForceLoggingClause
+    : (ENABLE | DISABLE) FORCE (LOGGING | NOLOGGING)
+    | SET STANDBY NOLOGGING FOR ((DATA AVAILABILITY) | (LOAD PERFORMANCE))
+    ;
+
+pdbRefreshModeClause
+    : REFRESH MODE (MANUAL | (EVERY refreshInterval (MINUTES | HOURS)) | NONE )
+    ;
+
+pdbRefreshSwitchoverClause
+    : FROM sourcePdbName AT_ dbLink SWITCHOVER
+    ;
+
+pdbDatafileClause
+    : pdbName? DATAFILE (fileNameAndNumber | ALL)  (ONLINE | OFFLINE)
+    ;
+
+fileNameAndNumber
+    : (fileName | fileNumber) (COMMA_ (fileName | fileNumber))*
+    ;
+
+pdbRecoveryClauses
+    : pdbName? (pdbGeneralRecovery
+    | BEGIN BACKUP
+    | END BACKUP
+    | ENABLE RECOVERY
+    | DISABLE RECOVERY)
+    ;
+
+pdbGeneralRecovery
+    : RECOVER AUTOMATIC? (FROM locationName)? (DATABASE
+    | TABLESPACE tablespaceName (COMMA_ tablespaceName)*
+    | DATAFILE fileNameAndNumber
+    | LOGFILE fileName
+    | CONTINUE DEFAULT?)?
+    ;
+
+pdbChangeState
+    : pdbName? (pdbOpen | pdbClose | pdbSaveOrDiscardState)
+    ;
+
+pdbOpen
+    : OPEN (((READ WRITE) | (READ ONLY))? RESTRICTED? FORCE?
+    | (READ WRITE)? UPGRADE RESTRICTED?
+    | RESETLOGS) instancesClause?
+    ;
+
+instancesClause
+    : INSTANCES EQ_ (instanceNameClause | (ALL (EXCEPT instanceName)?))
+    ;
+
+instanceNameClause
+    : LP_ instanceName (COMMA_ instanceName )* RP_
+    ;
+
+pdbClose
+    : CLOSE ((IMMEDIATE? (instancesClause | relocateClause)?) | (ABORT? instancesClause?))
+    ;
+
+relocateClause
+    : RELOCATE (TO instanceName)?
+    | NORELOCATE
+    ;
+
+pdbSaveOrDiscardState
+    : (SAVE | DISCARD) STATE instancesClause?
+    ;
+
+pdbChangeStateFromRoot
+    : (pdbNameClause | (ALL (EXCEPT pdbNameClause)?)) (pdbOpen | pdbClose | pdbSaveOrDiscardState)
+    ;
+
+pdbNameClause
+    : pdbName (COMMA_ pdbName)*
+    ;
+
+applicationClauses
+    : APPLICATION ((appName appClause) | (ALL SYNC))
+    ;
+
+appClause
+    : BEGIN INSTALL SQ_ appVersion SQ_ (COMMENT SQ_ commentValue SQ_)?
+    | END INSTALL (SQ_ appVersion SQ_)?
+    | BEGIN PATCH NUMBER_ (MINIMUM VERSION SQ_ appVersion SQ_)? (COMMENT SQ_ commentValue SQ_)?
+    | END PATCH NUMBER_?
+    | BEGIN UPGRADE (SQ_ startAppVersion SQ_)? TO SQ_ endAppVersion SQ_ (COMMENT SQ_ commentValue SQ_)?
+    | END UPGRADE (TO SQ_ endAppVersion SQ_)?
+    | BEGIN UNINSTALL
+    | END UNINSTALL
+    | SET PATCH NUMBER_
+    | SET VERSION SQ_ appVersion SQ_
+    | SET COMPATIBILITY VERSION ((SQ_ appVersion SQ_) | CURRENT)
+    | SYNC TO ((SQ_ appVersion SQ_) | (PATCH patchNumber))
+    | SYNC
+    ;
+
+snapshotClauses
+    : pdbSnapshotClause
+    | materializeClause
+    | createSnapshotClause
+    | dropSnapshotClause
+    | setMaxPdbSnapshotsClause
+    ;
+
+pdbSnapshotClause
+    : ENABLE SNAPSHOT (MANUAL | (EVERY snapshotInterval (HOURS | MINUTES)) | NONE)
```

Comment:
And in snapshotClauses, there also have differences: 
- in graphic syntax diagrams:
<img width="265" alt="image" src="https://user-images.githubusercontent.com/60494790/180858069-3deed5cf-96b9-4627-8754-08ff0eb0257a.png">

- in the description of image, there is an additional `ENABLE` keyword:
<img width="488" alt="38acb62dff94d19d531e2f635fe450a" src="https://user-images.githubusercontent.com/60494790/180858156-c4e4ca89-a6b0-4ffc-aaae-a055601d23e3.png">

Reply:
I think you can execute these sqls in Oracle database.

Reply:
Thanks, I will check these in Oracle.

Reply:
According to the SQL execution result, we should remove the `ENABLE` keyword.

<img width="389" alt="5e87aeebbc0b2261b26d42b17526f85" src="https://user-images.githubusercontent.com/60494790/181415381-cbfdd454-0a08-4ad3-891b-1a60c6e84808.png">